### PR TITLE
New package: python-scour-0.35

### PR DESCRIPTION
**Pull request by o1ux**

### CODE REVIEW

File: srcpkgs/scour/template [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# Template file for 'scour'
+pkgname=scour
+version=0.35
+revision=1
+noarch=yes
+build_style=python-module
+hostmakedepends="python-setuptools python3-setuptools"
+depends="python-setuptools python-six"
```
Comment (Vaelatern) [inline srcpkgs/scour/template]: Does it really depend on setuptools?

File: srcpkgs/scour/template [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+short_desc="An SVG scrubber (Python2)"
+maintainer="Alexander Mamay <alexander@mamay.su>"
+license="Apache-2.0"
+homepage="https://github.com/scour-project/scour"
+distfiles="$homepage/archive/v${version}.tar.gz"
+checksum=91a7dd0d721a3567b802c1b353ef451621594e628fe723cedb65cd4629fbc96f
+
+python3-scour_package() {
+	noarch=yes
+	depends="python3-setuptools python3-six"
```
Comment (Vaelatern) [inline srcpkgs/scour/template]: Or here?

Comment (o1ux) [inline srcpkgs/scour/template]: Without python-setuptools:
```
$ scour2 
Traceback (most recent call last):
  File "/usr/bin/scour2", line 6, in <module>
    from pkg_resources import load_entry_point
ImportError: No module named pkg_resources
```

**Comment by Vaelatern** [comment]
Please set the variable `pycompile_module` for the package and subpackage.
`pycompile_module="scour"`

**Comment by Vaelatern** [comment]
Also please set `alternatives` akin to those used in `youtube-dl`

File: srcpkgs/scour/template [hunk 3]
```diff
@@ -0,0 +1,24 @@
+# Template file for 'scour'
+pkgname=scour
```
Comment (asergi) [inline srcpkgs/scour/template]: `pkgname=python-scour`

**Pull request merged**